application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -117,6 +117,11 @@ private String getMessageContent(Message message) {
         if (message.getEmbeds().isEmpty()) {
             return message.getContentRaw();
         }
-        return message.getEmbeds().get(0).getDescription();
+
+        StringBuilder messageContent = new StringBuilder();
+        message.getEmbeds()
+            .forEach(embed -> messageContent.append("%s%n".formatted(embed.getDescription())));
```

Comment:
works. but a bit more idiomatic would be to use streams
```java
message.getEmbeds().stream()
  .collect(Collectors.joining("\n"))
```
(also note that its necessary to hardcode the line separator to `\n` here, since that is what discord uses. if u go platform-dependent, u would get `\r\n` on windows, which discord would be confused about